packages/nova-types/src/nova-localization.ts

```diff
@@ -0,0 +1,29 @@
+const LocalizedString = Symbol("LocalizedString");
+
+export type StringWithPlaceholders<T extends Record<string, string | number>> =
+  {
+    [LocalizedString]: T;
```

Comment:
Is this intentional that the symbol `LocalizedString` is not exported? A way to have a newtype to hide the actual, irrleevant, way this type is constructed?

Reply:
Yes, it makes it more difficult for folks to work around it by manually creating something. 

It would still be possible to cast though, so that workaround is always available